website/docs/r/cm_network.markdown

```diff
@@ -0,0 +1,54 @@
+---
+layout: "rightscale"
+page_title: "Rightscale: network"
+sidebar_current: "docs-rightscale-resource-network"
+description: |-
+  Create and maintain a rightscale cloud management network.
+---
+
+# rightscale_network
+
+Use this resource to create, update or destroy rightscale [networks](http://docs.rightscale.com/cm/dashboard/manage/networks/index.html) in cloud management.
+
+## Example Usage
+
+```hcl
+resource "rightscale_network" "us-oregon-devops-vpc" {
+  name = "us-oregon-devops-vpc"
+  description = "AWS US Oregon vpc for devopery"
+  cloud_href = "/api/clouds/6"
+  cidr_block = "192.168.0.0/16"
+}
+
+output "us-oregon-devops-vpc-uuid" {
+  value = "${rightscale_network.us-oregon-devops-vpc.resource_uid}"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `cloud_href` - (Required) Cloud you want to create the network in.
```

Comment:
Would be nice to explain how to retrieve these.

Reply:
As in how to define an output in tf?  I can do that but its a top level datasource construct (https://github.com/rightscale/terraform-provider-rightscale/blob/master/website/docs/d/cm_cloud.markdown#example-usage) that we use over and over and over again so I go back and forth on how much linking or extra examples to do vs. assuming developer folks will quickly realize they always need to define a few things, like cloud, datacenter, instance type, etc. to do anything meaningful with this toolset.

Shall I add a link or example comment to the hcl example for this resource?

Reply:
yeah sounds good. I guess the comment ought to say `href of Cloud ...`